R/pkg/DESCRIPTION

```diff
@@ -1,28 +1,27 @@
 Package: SparkR
 Type: Package
-Title: R Frontend for Apache Spark
 Version: 2.1.0
-Date: 2016-11-06
```

Comment:
this is removed - I tried but haven't found a way to update this automatically, (I guess this could be in the [release-tag](https://github.com/apache/spark/blob/master/dev/create-release/release-tag.sh) script though)
But more importantly, seems like many (most?) packages do not have this in their DESCRIPTION.

In any case, release date are stamped when releasing to CRAN.